pages/common/acme.sh-dns.md

```diff
@@ -0,0 +1,25 @@
+# acme.sh --dns
+
+> Use a DNS-01 challenge to issue a certificate.
+> DNS API mode documentation: <https://github.com/acmesh-official/acme.sh/wiki/dnsapi>.
+> More information: <https://github.com/acmesh-official/acme.sh/wiki/DNS-manual-mode>.
+
+- Issue a certificate using an automatic DNS API mode:
+
+`acme.sh --issue --dns {{gnd_gd}} --domain {{example.com}}`
+
+- Issue a wildcard certificate (denoted by an asterisk) using an automatic DNS API mode:
+
+`acme.sh --issue --dns {{dns_namesilo}} --domain {{*.example.com}}`
```

Comment:
It's good practice here to also request that the certificate is valid for the root domain. With `certbot`, that would be like so:
```suggestion
`acme.sh --issue --dns {{dns_namesilo}} --domain  {{example.com}} --domain {{*.example.com}}`
```

Does this work for `acme.sh` too?

Reply:
Of course it does. Multiple domains support is available, I just did not find a place to document it nicely on in https://github.com/tldr-pages/tldr/pull/5823

As a side note, I have a wildcard for subdomains that does not share a certificate with the root domain (not residing on the same VPS), but I agree that is more of an edge case. Combining them both into the one certificate is probably fairly common practice.

Reply:
Thinking about it a little bit more, I really like this place to mention multiple domain certificate, because of the introduction of a mentioned good practice. 

However, since multiple domains topic is already open, shouldn't there be a mention in the parent as well?

Reply:
Included via https://github.com/tldr-pages/tldr/pull/5852/commits/5a7e01531e84ae34d62d2bc76271b6fcf3a2807c